app-backend/db/src/main/scala/com/azavea/rf/database/OrganizationDao.scala

```diff
@@ -215,23 +211,23 @@ object OrganizationDao extends Dao[Organization] with LazyLogging {
      )
   }
 
-  def setVisibility(visibility: Visibility, organizationId: UUID): ConnectionIO[Int] = {
-    val updateTime = new Timestamp((new java.util.Date()).getTime)
+  def setVisibility(visibility: Visibility): ConnectionIO[Int] = {
```

Comment:
How is this scoped to a single organization?

Reply:
@jisantuc Worried this is just a blanket `UPDATE` on the `organizations` table. Do you know if `tableF` is taking care of the scoping some how?